### PR TITLE
EDU-388 Support home page template injection

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-388

### CODE REVIEW

File: src/components/pages/index.js [hunk 1]
```diff
@@ -1,47 +1,27 @@
...
+function Index({ HomePageTemplate }) {
   const handleSearch = searchText => {
     window.location = urls.getSearchUrl(searchText);
   };
 
   const alerts = useGlobalAlerts();
 
   return (
-    <PageTemplate alerts={alerts} fullScreen>
+    <HomePageTemplate alerts={alerts} fullScreen>
```
Comment (ahelmberger) [inline src/components/pages/index.js:15]: I think as there is only one homepage, we do not need the distinction fullscreen=true/false ?

Comment (adrianaluchian) [reply src/components/pages/index.js:15]: Was thinking the same, but if no `HomePageTemplate` is set, we default to `DefaultPageTemplate` which requires `fullScreen`.
However we will pass in a HomePageTemplate anyway from both projects, so then I guess we could just simplify and drop `fullScreen`

Comment (adrianaluchian) [reply src/components/pages/index.js:15]: Scratch that, I got it wrong 🤦‍♀️ Will remove `fullScreen`

Comment (ahelmberger) [reply src/components/pages/index.js:15]: Ah, I see. Then I'll leave that decision to your taste :-)

**Review by ahelmberger — APPROVED** [review]

File: src/components/pages/index.js (hunk 1 above)
Comment (ahelmberger) [reply src/components/pages/index.js:15]: Ok, too late :laughing:

Comment (adrianaluchian) [reply src/components/pages/index.js]: 😅 thanks! Updated though!

**Pull request merged**